src/status_im/network/core.cljs

```diff
@@ -190,28 +190,9 @@
         {:ui/show-error "chain-id already defined"}))
     {:ui/show-error "invalid network parameters"}))
 
-(defn- navigate-to-network-details
-  [cofx network show-warning?]
-  (fx/merge cofx
-            (when show-warning?
-              {:utils/show-popup {:title   "LES support is experimental!"
-                                  :content "Use at your own risk!"}})
-            (navigation/navigate-to-cofx :network-details {:networks/selected-network network})))
-
-(defn- not-supported-warning [fleet]
-  (str (name fleet) " does not support LES!\n"
-       "Please, select one of the supported fleets:"
-       (map name fleet-core/fleets-with-les)))
-
 (fx/defn open-network-details
   {:events [::network-entry-pressed]}
   [cofx network]
   (let [db                  (:db cofx)
         rpc-network?        (get-in network [:config :UpstreamConfig :Enabled] false)
-        fleet               (fleet-core/current-fleet db)
-        fleet-supports-les? (fleet-core/fleet-supports-les? fleet)]
-    (if (or rpc-network? fleet-supports-les?)
-      (navigate-to-network-details cofx network (not rpc-network?))
```

Comment:
maybe we should keep rpc-network? and navigate-to-network-details  ?

Reply:
Ah, you might be right, but I don't really know how to do that, I can't really write Clojure. I barely managed to get this PR working because removing stuff is easier than adjusting/adding. Could you help me out?

Reply:
yes sure, i'll take a look a little bit later today